static/inline/loader.js

```diff
@@ -0,0 +1,174 @@
+//The selfs loader, so far it takes care of the aspect ratio of the iframes,
```

Comment:
Latest "widget loading" approach does not longer require _static/inline/loader.js_
Everything shall happen on _static/inline/index.js_